resources/public/api/conf/1.0/common/errors.yaml

```diff
@@ -0,0 +1,94 @@
+components:
+  examples:
+
+    # 400
+
+    formatNino:
+      description: The format of the supplied NINO field is not valid.
+      value:
+        code: FORMAT_NINO
+        message: The provided NINO is invalid
+
+    formatTaxYear:
+      description: The format of the supplied taxYear field is not valid.
+      value:
+        code: FORMAT_TAX_YEAR
+        message: The provided tax year is invalid
+
+    formatSource:
+      description: The format of the supplied Source field is not valid.
+      value:
+        code: FORMAT_SOURCE
+        message: The provided source is invalid
+
+    formatValue:
+      description: One or more values have been added with the incorrect format.
+      value:
+        code: FORMAT_VALUE
+        message: The field should be between 0 and 99999999999.99
+
+    formatCustomerRef:
+      description: The format of customer reference is not valid.
+      value:
+        code: FORMAT_CUSTOMER_REFERENCE
+        message: The provided customer reference is not valid
+
+    ruleIncorrectOrEmptyBody:
+      description: An empty or non-matching body was submitted.
+      value:
+        code: RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED
+        message: An empty or non-matching body was submitted
+
+    ruleTaxYearNotSupported:
+      description: |
```

Comment:
For the documentation scenario on this error in API #1668, it is `The specified tax year is not supported. That is, the tax year specified is before the minimum tax year value.` Also, the error message is `The tax year specified is before the minimum tax year value`. Do we just want to use one (as presumably we just have one in the code) or separate documentation errors?

Reply:
This also applies to API #1669